install_config/aggregate_logging.adoc

```diff
@@ -1354,28 +1416,42 @@ Curator. Currently the `logging-curator` ConfigMap is used to configure both
 your ops and non-ops Curator instances. Any `.operations` configurations are
 in the same location as your application logs configurations.
 
-. To edit the provided ConfigMap to configure your Curator instances:
+. To create the Curator configuration, edit the configuration in the deployed
+ConfigMap:
 +
 ----
 $ oc edit configmap/logging-curator
 ----
 
-. To replace the provided ConfigMap instead:
+. For scripted deployments, copy the configuration file that was created by the
+installer and create your new OpenShift custom config:
 +
 ----
-$ create /path/to/mycuratorconfig.yaml
-$ oc create configmap logging-curator -o yaml \
-  --from-file=config.yaml=/path/to/mycuratorconfig.yaml | \
-  oc replace -f -
+$ oc extract configmap/logging-curator --keys=curator5.yaml,config.yaml --to=/my/config
+$ edit /my/config/curator5.yaml
```

Comment:
edit is not one command
shall we delete the "$"?

Reply:
I updated this. Is it correct?